robot_smach_states/src/robot_smach_states/navigation/navigation.py

```diff
@@ -103,6 +104,10 @@ def execute(self, userdata=None):
         if self.reset_head:
             self.robot.head.close()
 
+        # Move the robot to a suitable driving pose
+        if self.reset_pose and self.robot.base.global_planner.path_length > 0.5:
```

Comment:
Hmmm this magic number is not really nice here, however I can not really think of a better solution so I guess leave it in for now?